zubhub_frontend/zubhub/src/assets/js/styles/components/button/buttonStyles.js

```diff
@@ -1,7 +1,7 @@
 const styles = theme => ({
   primaryButtonStyle: {
     backgroundColor: '#00B8C4',
-    borderRadius: 30,
+    borderRadius: '30 !important',
```

Comment:
We should try to avoid `!important` As much as poss.  And if you’re trying to pass a string you need to pass the units I believe like 30px